lib/pulsar/client/config_util.py

```diff
@@ -58,9 +58,6 @@ def __find_type(path, explicit_type, default_type):
 
```

Comment:
Reading line-by-line :bird: :eyes: 

This file had no `from galaxy import eggs` removed, so it looked suspicious.